app/models/site.rb

```diff
@@ -125,6 +125,14 @@ def stylesheet_link
     end
   end
 
+  def og_image
+    hero_image&.opengraph&.url ? hero_image.opengraph.url : false
```

Comment:
Would it be better to return a default url than `false`. In the case of false, what happens when the tag is rendered? 

Same question for `og_description`.

Reply:
This is set globally in the `_meta.html.erb` partial and not per-model :)